build/x86_64_bionic/Dockerfile

```diff
@@ -18,12 +18,10 @@ RUN apt-get update -y \
         libcurl4-openssl-dev \
         libhiredis-dev \
         libltdl-dev \
-        libmongoc-dev \
         libmysqlclient-dev \
         libpq-dev \
```

Comment:
These are only needed for PostgreSQL, and can be removed. Also in other build `Dockerfile`s, `control` files, and `rules`.

Reply:
Good catch! Removed.